### PR TITLE
fix: shortening invite url

### PR DESCRIPTION
**Pull request by MatriQ**
Shortening invite url from `/activate?workspace_id=****&email=****&token=***` to  `/activate?token=***`, 

The old pattern of invite url is still supported to activate.

### CODE REVIEW

File: api/services/account_service.py [hunk 1]
```diff
@@ -401,7 +406,7 @@ def invite_new_member(cls, tenant: Tenant, email: str, role: str = 'normal',
         # send email
         send_invite_member_mail_task.delay(
             to=email,
-            token=cls.generate_invite_token(tenant, account),
```
Comment (MatriQ) [inline api/services/account_service.py:404]: Is it a bug? I'm not sure why it uses different tokens between the front end and the invite email.

**Review by crazywoola — COMMENTED** [review]
LGTM

File: api/services/account_service.py (hunk 1 above)
Comment (crazywoola) [reply api/services/account_service.py:404]: Black history

Comment (takatost) [reply api/services/account_service.py:404]: yep😅

**Review by crazywoola — DISMISSED** [review]

**Comment by MatriQ** [comment]
Based on the discussion with @takatost , added the email to the returned invite URL:
604662ef94dbd4f884a93783279b871

**Review by takatost — APPROVED** [review]
thx!

**Pull request merged**